types/npm-license-crawler/index.d.ts

```diff
@@ -4,13 +4,15 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.2
```

Comment:
@ffflorian Is it still need header?

Reply:
If it works without it, you can remove it.